packages/css/src/components/card/card.scss

```diff
@@ -60,3 +66,9 @@
     outline: none;
   }
 }
+
+.ams-card--horizontal {
+  @include ams-grid;
+
+  padding-inline: 0 !important;
+}
```

Comment:
Let’s find a cool approach for horizontal Cards. Refine the approach to apply a grid to it? Use container queries for fluid layout?

Reply:
> You don't need a grid in card to allow it to become horizontal. It adds too much markup imho.

I agree. One goal of implementing these page designs is to find new aspects – and that’s whats happening here. The Card Image has been designed to span four columns, and this was the easiest approach with our current components. The next step is to either find a more effient implementation, or adapt the design. A spike for this is on our sprint.

> Beside that it should not be in this PR probably.

It was still a draft on purpose, but I’ll replace ‘feat’ with ‘poc’ in the title to clarify.